pkg/exchange/bybit/bybitapi/get_open_order_request.go

```diff
@@ -0,0 +1,159 @@
+package bybitapi
+
+import (
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/c9s/requestgen"
+)
+
+//go:generate -command GetRequest requestgen -method GET -responseType .APIResponse -responseDataField Result
+//go:generate -command PostRequest requestgen -method POST -responseType .APIResponse -responseDataField Result
+
+type OpenOrders struct {
```

Comment:
Could we rename to OpenOrdersResponse to avoid confusion?

Reply:
ok